src/librustc/diagnostics.rs

```diff
@@ -279,6 +279,41 @@ println!("{}", Y);
 ```
 "##,
 
+E0019: r##"
+A function call isn't allowed in the const's initialization expression
+because the expression's value must be known at compile-time. Bad example:
```

Comment:
I actually had the same comment on the other PRs I opened yesterday. I forgot to update this one, I do it.